src/app/components/app-layout/app-layout.component.scss

```diff
@@ -200,6 +201,17 @@
     color: #f8f8f8;
     text-transform: capitalize;
   }
+
+  // Move button to the bottom left and make it vertical
+  @media (max-width: 768px) {
+    bottom: 30%;
+    left: 0;
+
+    .feedback-button {
+      transform: rotate(90deg);
+      transform-origin: left bottom;
+    }
```

Comment:
I suggest to use `@include media-breakpoint-down(md) { ... }` instead of `@media (max-width: 768px) { ... }` because this way we handle media queries over all the different files, the same way.

Also modifying the bootstrap breakpoints will update the col classes from bootstrap and your breakpoint at the same time without worrying to update your breakpoint manually whenever you update the bootstrap breakpoints.